docs/xapi-extensions/eventVersion.rst

```diff
@@ -0,0 +1,5 @@
++-------------------------------------------------------------------------------------------------------+--------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Extensions                                                                                            | Title              | Description                                                                                                                                                                                                                                                                    |
++=======================================================================================================+====================+================================================================================================================================================================================================================================================================================+
+| https://github.com/edly-io/event-routing-backends/blob/ERTE-108/docs/xapi-extensions/eventVersion.rst | "event version"@en | "This version represents version of mapping of a single edx event onto xapi format. Value should be a string of format "X.Y" where increment in X represents breaking changes and increment in Y represents addition of fields in the event json, both starting with 0 (zero)" |
```

Comment:
@ayub02 can we remove `ERTE-108` from resource uri?

Reply:
@ziafazal I can replace it with `master`. That way it will be resolvable one the PR is merged. Sound good?